src/commonMain/kotlin/file.kt

```diff
@@ -13,6 +13,10 @@ expect class File(rawPath: String) {
     val isRoot: Boolean
     fun ls(): List<File>
     fun lines(): List<String>
+    fun move(dest:File): Boolean
```

Comment:
Since the removal functions are called `rm` and `rmdir`, and listing is `ls`, for consistency let's make this one `mv`. Yay *nix!